source/_integrations/history_stats.markdown

```diff
@@ -121,7 +121,11 @@ Depending on the sensor type you choose, the `history_stats` integration can sho
 
 - **time**: The default value, which is the tracked time, in hours
 - **ratio**: The tracked time divided by the length of your period, as a percentage
-- **count**: How many times the tracked entity matched the configured state during the time period
+- **count**: How many times the tracked entity matched the configured state during the time period. Unlike a rising edge counter, this will also include a count if the entity is already in the matched state at the beginning of the time period.
+
+{% note %}
+For a count sensor with a time period that does not overlap (such as one that resets hourly, daily, or monthly), consider using [customization](/docs/configuration/customizing-devices/#customizing-an-entity-in-yaml) to change the `state_class` to `total_increasing` to generate useful statistics.
```

Comment:
`For a count sensor with a time period that does not overlap`
 this references a target entity (as defined by the entity_id of this sensor?  What would be an example of that? A step counter like FitBit?
`that does not overlap`
 overlap with what?

`to change the state_class to total_increasing to generate useful statistics`
is this referring to the state_class of this (history_stat) entity ?

Reply:
> `For a count sensor with a time period that does not overlap`
this references a target entity (as defined by the entity_id of this sensor? What would be an example of that? A step counter like FitBit?

This note applies to any history_stats sensor that is set to `type: count`. Which would be anything you want to count, like how many times a light is `on` in a day. 

> `that does not overlap`
overlap with what?

I will work on how to clarify this. I meant it doesn't overlap with itself, meaning it isn't a sliding time window (e.g. a trailing 60 minute window). It would instead be a time window whose duration increases while the start time is fixed, and once the duration reaches a maximum value the start time will jump, and the cycle repeats. For example, a time window that starts at `today_at('0:00:00')` and ends at `now()`. Essentially, any type of window that would result in a sensor the acts like a utility_meter with a specified reset cycle. 

> `to change the state_class to total_increasing to generate useful statistics`
is this referring to the state_class of this (history_stat) entity ?

Yes. A shortcoming with this integration is that it cannot detect how the user set up the time window and therefore cannot set the `state_class` of the new sensor accordingly. So I believe that the documentation should make the user aware of this so the user can take action themselves.

Reply:
Reworded the description of the time window to be more clear.